tests/test_pytypes.cpp

```diff
@@ -662,8 +662,8 @@ TEST_SUBMODULE(pytypes, m) {
 // This is "most correct" and enforced on these platforms.
```

Comment:
But why is it now also indenting this?
Just wondering. Maybe we get more free spaces here in the next upgrade?